x-pack/plugins/snapshot_restore/public/application/components/policy_form/steps/step_settings/step_settings.tsx

```diff
@@ -0,0 +1,206 @@
+              defaultMessage="Ignore unavailable indices"
+            />
+          </h3>
+        </EuiTitle>
+      }
+      description={
+        <FormattedMessage
+          id="xpack.snapshotRestore.policyForm.stepSettings.ignoreUnavailableDescription"
+          defaultMessage="Ignores indices that are unavailable when taking the snapshot. Otherwise, the entire snapshot will fail."
+        />
+      }
+      fullWidth
+    >
+      <EuiFormRow hasEmptyLabelSpace fullWidth>
+        <EuiSwitch
+          data-test-subj="ignoreUnavailableIndicesToggle"
+          label={
+            <FormattedMessage
+              id="xpack.snapshotRestore.policyForm.stepSettings.ignoreUnavailableLabel"
+              defaultMessage="Ignore unavailable indices"
+            />
+          }
+          checked={Boolean(config.ignoreUnavailable)}
+          onChange={(e) => {
+            updatePolicyConfig({
+              ignoreUnavailable: e.target.checked,
+            });
+          }}
+        />
+      </EuiFormRow>
+    </EuiDescribedFormGroup>
+  );
+
+  const renderPartialField = () => (
+    <EuiDescribedFormGroup
+      title={
+        <EuiTitle size="s">
+          <h3>
+            <FormattedMessage
+              id="xpack.snapshotRestore.policyForm.stepSettings.partialDescriptionTitle"
+              defaultMessage="Allow partial indices"
+            />
+          </h3>
+        </EuiTitle>
+      }
+      description={
+        <FormattedMessage
+          id="xpack.snapshotRestore.policyForm.stepSettings.partialDescription"
+          defaultMessage="Allows snapshots of indices with primary shards that are unavailable. Otherwise, the entire snapshot will fail."
+        />
+      }
+      fullWidth
+    >
+      <EuiFormRow hasEmptyLabelSpace fullWidth>
+        <EuiSwitch
+          data-test-subj="partialIndicesToggle"
+          label={
+            <FormattedMessage
+              id="xpack.snapshotRestore.policyForm.stepSettings.partialIndicesToggleSwitch"
+              defaultMessage="Allow partial indices"
+            />
+          }
+          checked={Boolean(config.partial)}
+          onChange={(e) => {
+            updatePolicyConfig({
+              partial: e.target.checked,
+            });
+          }}
+        />
+      </EuiFormRow>
+    </EuiDescribedFormGroup>
+  );
+
+  const renderIncludeGlobalStateField = () => (
+    <EuiDescribedFormGroup
+      title={
+        <EuiTitle size="s">
+          <h3>
+            <FormattedMessage
+              id="xpack.snapshotRestore.policyForm.stepSettings.includeGlobalStateDescriptionTitle"
+              defaultMessage="Include global state"
+            />
+          </h3>
+        </EuiTitle>
+      }
+      description={
+        <FormattedMessage
+          id="xpack.snapshotRestore.policyForm.stepSettings.includeGlobalStateDescription"
+          defaultMessage="Stores the global state of the cluster as part of the snapshot."
```

Comment:
Nit: I think `cluster state` is a pretty common phrase in the ES docs.
```suggestion
          defaultMessage="Stores the global cluster state as part of the snapshot."
```

On a broader note, it may be helpful to somehow mention/present the resources/info included in the global cluster state.